setup.py

```diff
@@ -8,11 +8,16 @@
 setup(
     name='awsiotsdk',
     version='1.0.0-dev',
-    license='Apache-2.0',
+    license='Apache Software License',
```

Comment:
Maybe this should remain as "Apache-2.0'?
Honestly, this all baffles me 😐
Ask a lawyer?

Here are some github issues where people are just as confused as us regarding what they're supposed to do, because "Apache Software License" is less specific than "Apache-2.0" but it's the only classifier available:
https://github.com/pypa/pypi-legacy/issues/91
https://github.com/pypa/trove-classifiers/issues/17

And here's an issue about how the `classifier` field is recommended in the dev guide, but the APIs still use the `license` field:
https://github.com/pypa/warehouse/issues/8960

Reply:
Different communities have different standardized names for common licenses. The list of valid classifiers for PyPI is at https://pypi.org/classifiers/ and you can search for `Apache` there to see which is preferred.